src/backbone.marionette.js

```diff
@@ -58,12 +58,12 @@ Marionette.noConflict = function() {
 };
 
 // Utilities
-Marionette.bindEntityEvents = proxy(bindEntityEvents);
-Marionette.unbindEntityEvents = proxy(unbindEntityEvents);
-Marionette.bindRadioEvents = proxy(bindEntityEvents);
-Marionette.unbindRadioEvents = proxy(unbindEntityEvents);
-Marionette.bindRadioRequests = proxy(bindRadioRequests);
-Marionette.unbindRadioRequests = proxy(unbindRadioRequests);
+Marionette.bindEvents = proxy(bindEvents);
+Marionette.unbindEvents = proxy(unbindEvents);
+Marionette.bindRadioEvents = proxy(bindEvents);
```

Comment:
I think it is perfectly ok to remove `bindRadioEvents` here as it matches the renaming of `bindRadioRequests`